app/views/gitter/chat_button.erb

```diff
@@ -0,0 +1,17 @@
+<% if control.language.downcase == 'ruby' %>
+  <div class="gitter-chat-button-frame">
+    <div class="gitter-chat-button">
+      Open Chat: Enjoy the <%= control.language.capitalize %> conversation!
+    </div>
+  </div>
+
+  <script>
+    ((window.gitter = {}).chat = {}).options = {
+      room: 'exercism/xruby',
```

Comment:
I believe so, but since we have that encoded into the track's `config.json` the rest of the system shouldn't have to know this. We can rely directly on the value that is passed to us by x-api. https://github.com/exercism/xruby/blob/master/config.json#L6